cmd2/cmd2.py

```diff
@@ -130,7 +133,9 @@ def __init__(self, completekey: str = 'tab', stdin=None, stdout=None, *,
                  startup_script: str = '', use_ipython: bool = False,
                  allow_cli_args: bool = True, transcript_files: Optional[List[str]] = None,
                  allow_redirection: bool = True, multiline_commands: Optional[List[str]] = None,
-                 terminators: Optional[List[str]] = None, shortcuts: Optional[Dict[str, str]] = None) -> None:
+                 terminators: Optional[List[str]] = None, shortcuts: Optional[Dict[str, str]] = None,
+                 command_sets: Optional[Iterable[CommandSet]] = None,
+                 auto_load_commands: bool = True) -> None:
```

Comment:
It would only affect your code if you modified it to add support for this. Legacy code would be unaffected because all of this is new and didn't exist before. The goal was to make the default case easier to use.